### ISSUE TITLE
Inventory Count is Doubled even if storage container only has one Connection

### ISSUE THREAD

**Issue by Morabert** [PrinceOfAmber/Storage-Network#399]
Minecraft Version: Java (is there a Bedrock port?)

Forge Version: 36.2.5

Mod Version: 1.4.3

Single Player or Server: Single Player

Describe problem (what you were doing; what happened; what should have happened):

Storage Count Doubles even when only one cable is connected to Storage Container

Log file link:

Video/images/gifs (direct upload or link): 

![ay](https://user-images.githubusercontent.com/36409264/139707380-629c92b0-77b4-4c85-bfc1-8933e2c38ac6.JPG)

![ay](https://user-images.githubusercontent.com/36409264/139707216-2b17de2d-ad2b-403a-bbc2-fcb7ce4aa220.JPG)
![ceee](https://user-images.githubusercontent.com/36409264/139707219-bb0eb067-b7d5-45ab-9880-3f5b53e9e12f.JPG)
![bee](https://user-images.githubusercontent.com/36409264/139707221-16b1fe0a-ba8e-4932-b47e-25a865bb2137.JPG)

**Comment by haydo75332** [comment]
I'm having the same problem as well

**Comment by Lothrazar** [comment]
Yeah dont connect two cables to the same inventory.  Nothing gets duplicated its just displaying them twice.

You can do this with a single 1x1 chest and connect a bunch of cables to it

**Comment by Morabert** [comment]
The containers have two cables because the way the way the mod descriptions are worded it suggests that both an input & an output cable are needed as separate, distinct, entities.

If for some reason three separate people were to put their hands into my trouser pockets that would not, by sheer force of query attempt, instantaneously triple the amount of money in my wallet.

If the system is counting _per connection_ & not _per storage container_, 
that would be a fundamental error in the order of syntax that can & will compound in to greater & greater compounded complications.

**Comment by Oddlybird** [comment]
I'm also having this problem on 16.5, version 1.5.3.  I'm using a single link cable as the only connection to an Obsidian Barrel (from iron barrels mod), and another single link cable is the only link to a Storage Drawers 2x2 drawer.  All item numbers are doubled, or more.
